### ISSUE TITLE
[Feature Request] Support grayscale images for classifier on Android

### ISSUE THREAD

**Issue by khomutovsky** [tensorflow/tensorflow#17053]
How can one modify ClassifierActivity.java (https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/android/src/org/tensorflow/demo/ClassifierActivity.java) to convert images obtained from the camera to grayscale and then pass them to the inferenceInterface?

**Have I written custom code:** I want to use the ClassifierActivity with another neural network that was trained on grayscale images
**OS Platform and Distribution:** N/A
**TensorFlow installed from:** pip
**TensorFlow version:** 1.2.0
**Python version:** 3.6
**Bazel version (if compiling from source):** N/A
**GCC/Compiler version (if compiling from source):** N/A
**CUDA/cuDNN version:** N/A
**GPU model and memory:** N/A
**Exact command to reproduce:** N/A

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?
